lib/inspec/cli.rb

```diff
@@ -1,19 +1,33 @@
 # Copyright 2015 Dominik Richter
 
-require 'logger'
-require 'thor'
-require 'json'
-require 'pp'
-require 'inspec/utils/json_log'
-require 'inspec/utils/latest_version'
-require 'inspec/base_cli'
-require 'inspec/plugin/v1'
-require 'inspec/plugin/v2'
-require 'inspec/runner_mock'
-require 'inspec/env_printer'
-require 'inspec/schema'
-require 'inspec/config'
+require 'inspec/utils/deprecation/deprecator'
 require 'inspec/dist'
+require 'inspec/backend'
+require 'inspec/dependencies/cache'
+
+module Inspec # TODO: move this somewhere "better"?
```

Comment:
I fear overloading the word 'core'; but maybe something like `lib/inspec/core.rb`?

Reply:
I do to... doesn't have to happen in this commit. that's why I leave TODO's behind as I travel through the code.